cloudsql-mysql-plugin/src/e2e-test/java/io/cdap/plugin/CloudMySql/BQValidation.java

```diff
@@ -0,0 +1,154 @@
+package io.cdap.plugin.CloudMySql;
+
+import com.google.cloud.bigquery.TableResult;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.CloudMySqlClient;
+import org.junit.Assert;
+
+import java.io.IOException;
+import java.sql.*;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.Date;
+import java.util.List;
+
+/**
+ *  BQValidation
+ */
+
+public class BQValidation {
+    public static void main(String[] args) {
+//        validateBQAndDBRecordValues(String schema, String sourceTable, String targetTable)
```

Comment:
Remove the main method.

Reply:
I don't see any commits. How should I make sure its resolved ?

Push the changes first then when marking them as resolved add the commit link in the comment.